packages/flutter/lib/src/foundation/key.dart

```diff
@@ -17,7 +17,10 @@ import 'package:meta/meta.dart';
 ///
 /// Subclasses of [Key] should either subclass [LocalKey] or [GlobalKey].
 ///
-/// See also the discussion at [Widget.key].
+/// See also:
+///
+///  * The discussion at [Widget.key] for more information about how widgets use
```

Comment:
This should probably be: "* [Widget.key], which discusses how widgets use keys." to be more in line with our style?

Reply:
Same below

Reply:
OK, fixed.